index.php

```diff
@@ -23,22 +23,22 @@
 
     $data = file_get_contents($_FILES['data']['tmp_name']);
 
-    $file = fopen($thefiledata, 'w');
+    $file = fopen($thefiledata, 'wb');
```

Comment:
Ok for all the edits, thanks @oleksandrsavenko.

Just this one: in which situation does `wb` add something to `w` ?
I.e. is there a case where `w` wouldn't work and where `wb` would solve it?